examples/import_export/import-export-jwk-aes-static-key.js

```diff
@@ -0,0 +1,31 @@
+import { crypto } from "k6/x/webcrypto";
+
+ export default async function () {
+    try {
```

Comment:
Stupid question; why do this example is surrounded by a `try..catch`, while other (e.g. `import-export-jwk-aes-key.js`) isn't, if the operations are the same? 🤔

Reply:
It's the other way around, I think it's just copy-paste issue, I will adjust the `import-export-jwk-aes-key.js` to also have a `try...catch`, which won't harm :+1: